.github/workflows/deploy-benchmark-db.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: DB Benchmark Service
```

Comment:
We aren't just benchmarking the database.

Reply:
changed to `Benchmark Service`